docs/installing-vyper.rst

```diff
@@ -190,18 +190,18 @@ Vyper can be downloaded as docker image from dockerhub:
 To run the compiler use the `docker run` command:
 ::
 
-    docker run -v $(pwd):/code vyper /code/<contract_file.vy>
+    docker run -v $(pwd):/code ethereum/vyper /code/<contract_file.vy>
 
 Alternatively you can log into the docker image and execute vyper on the prompt.
 ::
 
     docker run -v $(pwd):/code/ -it --entrypoint /bin/bash vyper
-    root@d35252d1fb1b:/code# vyper <contract_file.vy>
+    root@d35252d1fb1b:/code# ethereum/vyper <contract_file.vy>
```

Comment:
Did you try this change in Docker?

Reply:
I got the wrong line - this change should have been made on the line above. Fixing now

Reply:
Fixed